WooCommerce/Classes/POS/Presentation/Item Search/POSProductSearchable.swift

```diff
@@ -3,18 +3,25 @@ import enum Yosemite.POSItemType
 import protocol Yosemite.POSSearchHistoryProviding
 
 @available(iOS 17.0, *)
+
 final class POSProductSearchable: POSSearchable {
+    internal let itemListType: ItemListType
```

Comment:
nit;
```suggestion
final class POSProductSearchable: POSSearchable {
   let itemListType: ItemListType
```

Reply:
Ah! The compiler wasn't happy with some of the changes around conformances, no longer an issue but I must have left that there! Fixed: https://github.com/woocommerce/woocommerce-ios/pull/15568/commits/67c0893c99cf510bc13f237c4e7856dfcb37b170